dwcj-engine/src/main/java/org/dwcj/CookieStorage.java

```diff
@@ -0,0 +1,51 @@
+package org.dwcj;
+
+import com.basis.bbj.proxies.BBjThinClient;
+import com.basis.startup.type.BBjException;
+import org.dwcj.environment.ObjectTable;
+import org.dwcj.environment.StringTable;
+import org.dwcj.exceptions.DwcjRuntimeException;
+
+/**
+ * Gives access to the browsers cookie storage and allows adding/reading/removing values from it.
+ */
+public class CookieStorage extends AbstractWebStorage {
```

Comment:
The class does not provide method to add cookies using different samesite policies